### ISSUE TITLE
Saving history issue

### ISSUE THREAD

**Issue by andreevdenis** [ruuvi/com.ruuvi.station#567]

**Comment by andreevdenis** [comment]
1.6.16

**Comment by markoaamunkajo** [comment]
1.6.16 User made clean install in the evening, added local sensor, checked in the morning.
I believe I see the exact same issue in Chart for this sensor.

[Sunshine_20220111-1049.csv](https://github.com/ruuvi/com.ruuvi.station/files/7843555/Sunshine_20220111-1049.csv)

![sunshine](https://user-images.githubusercontent.com/50437378/148878308-694adda8-6170-4383-86ae-832531be18e9.jpg)

**Comment by andreevdenis** [comment]
No. Everything is fine on your screenshot. The problem was with saving BT advertisments. App was updating latest measurements but no points were saved in history.

**Comment by markoaamunkajo** [comment]
OK moving this to done
